Gems/Terrain/Code/Source/Components/TerrainWorldComponent.h

```diff
@@ -43,28 +43,28 @@ namespace Terrain
         AZ_RTTI(TerrainWorldConfig, "{295844DB-20DD-45B2-94DB-4245D5AE9AFF}", AZ::ComponentConfig);
         static void Reflect(AZ::ReflectContext* context);
 
-        AZ::Vector3 m_worldMin{ 0.0f, 0.0f, 0.0f };
-        AZ::Vector3 m_worldMax{ 1024.0f, 1024.0f, 1024.0f };
+        float m_minHeight{ 0.0f };
+        float m_maxHeight{ 1024.0f };
         float m_heightQueryResolution{ 1.0f };
         float m_surfaceDataQueryResolution{ 1.0f };
 
-        static AZ::Outcome<void, AZStd::string> ValidateBounds(AZ::Vector3& minBounds, AZ::Vector3& maxBounds)
+        static AZ::Outcome<void, AZStd::string> ValidateHeight(float minHeight, float maxHeight)
         {
-            if (!minBounds.IsLessEqualThan(maxBounds))
+            if (minHeight > maxHeight)
             {
-                return AZ::Failure(AZStd::string("World bounds min must be less than max."));
+                return AZ::Failure(AZStd::string("Terrain min height must be less than max height."));
```

Comment:
According to the math here and in FloatRange::IsValid(), the valid case is less equal than. But I think it is fine to leave as it is.